docs/CameraSupport.rst

```diff
@@ -61,31 +62,38 @@ Camera Support
     print(
         "Any support is impossible without the samples.\nCurrently, |rpu-button-cameras| cameras have samples, with total count of |rpu-button-samples| unique samples. **Please contribute samples**!\n\n")
 
-    for make, models in unique_makes.items():
-        print(make)
-        print('=' * len(make), "\n")
+    with open('docs/cameras.csv', 'w', newline='') as csvfile:
+        csvwriter = csv.writer(csvfile, delimiter=',',
+                               quotechar='\"', quoting=csv.QUOTE_ALL)
```

Comment:
I agree that table is probably better, but i'm not sure where that file will be placed,
so can we just keep the `print()`s please? I think this should work:
```
        csvwriter = csv.writer(sys.stdout, delimiter=',',
                               quotechar='\"', quoting=csv.QUOTE_ALL)
```

Reply:
Printing a table in `.rst` is a PITA. The point of the CSV file (that is not commited, as per `.gitignore` rule, by the way) is to be able to use 

```
.. csv-table:: Supported cameras
   :file: cameras.csv
   :header-rows: 1
```

Previously, I tried using native RST table with prints (https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html) but I couldn't make it work because apparently it needs the `-` marks to be perfectly aligned and Python prints make it very hard.

The CSV file is placed in the `/build/docs` folder when Sphinx is called from `cmake`.

Reply:
Aha.
Can you not just call `sys.stdout.write(, "   ")` before each `csvwriter.writerow()`?

Reply:
I'm not sure we understand each other. Either I print each line of the table with proper RST syntax so it gets (hopefully) parsed the right way, or I write the rows to a CSV file and let the RST parser use it in one shot. Why would I print and write to CSV ?